padle/padle.py

```diff
@@ -201,11 +233,17 @@ async def start(self, ctx):
                                        f" use `{prefix}padle resend`.",
                            color=discord.Color.teal())
         if await self.can_play_in_guild(ctx):
+            if len(await self.config.user(ctx.author).all_guesses()) == 0:
+                await ctx.send("Hello! It looks like this is your first PADle game.")
+                await ctx.send(self._get_help_text(ctx))
```

Comment:
you COULD send these help texts as a SimpleText menu2 if you want (this is optional though) - SimpleText is a precreated menu2 in tsutils like ClosableEmbed, for sending menus that are just text strings and nothing else